### PR TITLE
flatgeobuf: fix wkt srs

### PR DESCRIPTION
**Pull request by bjornharrtell**
Attempt to fix https://github.com/MapServer/MapServer/issues/6554.

### CODE REVIEW

File: mapflatgeobuf.c [hunk 1]
```diff
@@ -167,11 +167,13 @@ int msFlatGeobufLayerOpen(layerObj *layer)
...
     OGRSpatialReferenceH hSRS = OSRNewSpatialReference( NULL );
     char *pszWKT = NULL;
-    if (ctx->srid) {
+    if (ctx->srid > 0) {
       if (!(OSRImportFromEPSG(hSRS, ctx->srid) != OGRERR_NONE))
         return MS_FAILURE;
       if( OSRExportToWkt( hSRS, &pszWKT ) != OGRERR_NONE)
         return MS_FAILURE;
+    } else if (ctx->wkt == NULL) {
+      return MS_SUCCESS;
```
Comment (rouault) [inline mapflatgeobuf.c:176]: OSRDestroySpatialReference(hSRS); should be called here, and in the 2 above return MS_FAILURE cases

**Comment by jmckenna** [comment]
@bjornharrtell I updated `flatgeobuf.map` in this branch, so you can now run `map2img` locally and get the error (that is now also thrown on GitHub build action and Travis):
```
  map2img -m flatgeobuf.map -o ttt.png -all_debug 3
```
gives...
```
msDrawMap(): Image handling error. Failed to draw layer named 'africa-classes'. <br>
flatgeobuf_decode_header: (null) Failed to allocate buffer for header <br>
```

**Comment by jmckenna** [comment]
thanks @bjornharrtell works well here!  Nice to see the green build checkmarks ha.  Will give a couple hours before merging.  Thanks again!

**Comment by bjornharrtell** [comment]
Ok nice to hear it @jmckenna, I would suggest to squash merge.

**Comment by jmckenna** [comment]
thanks @bjornharrtell !   (will begin the beta2 process tomorrow...)

**Pull request merged**